tests/tests-tlr/src/test/java/edu/kit/kastel/mcse/ardoco/core/tests/integration/TraceabilityLinkRecoveryEvaluation.java

```diff
@@ -75,17 +83,60 @@ private void prepareCode(CodeProject codeProject) {
 
     protected abstract ImmutableList<String> getGoldStandard(CodeProject codeProject);
 
+    protected abstract ImmutableList<String> enrollGoldStandard(ImmutableList<String> goldStandard, ArDoCoResult result);
+
+    protected static ImmutableList<String> enrollGoldStandardForCode(ImmutableList<String> goldStandard, ArDoCoResult result) {
+        MutableList<String> enrolledGoldStandard = Lists.mutable.empty();
+
+        Model codeModel;
+        try {
+            codeModel = DataRepositoryHelper.getModelStatesData(result.dataRepository()).getModel(CodeModelType.CODE_MODEL.getModelId());
+            if (codeModel == null) {
+                logger.warn(WARNING_NO_CODE_MODEL);
+                return goldStandard;
+            }
+        } catch (NoSuchElementException e) {
+            logger.warn(WARNING_NO_CODE_MODEL);
+            return goldStandard;
+        }
+
+        for (String traceLink : goldStandard) {
+            enrolledGoldStandard.addAll(enrollTraceLink(codeModel, traceLink));
+        }
+
+        return enrolledGoldStandard.toImmutable();
+    }
+
+    private static List<String> enrollTraceLink(Model codeModel, String traceLink) {
+        MutableList<String> enrolledTraceLink = Lists.mutable.empty();
+
+        var splitTraceLink = traceLink.split(",");
+        var codeEntry = splitTraceLink[1].strip();
+        if (codeEntry.endsWith("/")) {
```

Comment:
This will always be a  `/` and not a `\` ? So it's hard coded in the code model module?

Reply:
The method is (currently) only called by the `enrollGoldStandardForCode` method (and is private). The gold standard only use `/` right now. Therefore, I think the assumption holds until someone forces (for whatever reason) other slashes at the gold standard or if someone uses the method for something else (but then should be knowledgeable if a private method is used. At best, we can add a warning or make the string a field or similar.